### PR TITLE
Updated Kotlin Version

### LINKED ISSUE #12: Update Kotlin version
We are trailing behind regarding the Kotlin version. Let's update!

### PR DESCRIPTION
**Pull request by hmajid2301**
Updated the kotlin version to 1.3.5 in `pom.xml`.

Fixes #12

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,7 @@ target
 
 .idea
 *.iml
+
+
+.history
+.vsocde
```
Comment (pvorb) [inline .gitignore]: Typo here

Also, please add a [newline at the end of the file](https://thoughtbot.com/blog/no-newline-at-end-of-file)

File: pom.xml [hunk 2]
```diff
@@ -347,4 +347,4 @@
 		</profile>
 	</profiles>
 
-</project>
+</project>
```
Comment (pvorb) [inline pom.xml]: Missing newline

File: pom.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 	<url>https://github.com/deviceinsight/helm-maven-plugin</url>
 
 	<properties>
-		<kotlin.version>1.3.11</kotlin.version>
+		<kotlin.version>1.3.5</kotlin.version>
```
Comment (pvorb) [inline pom.xml]: I don't see how this is a version upgrade. :wink:

Comment (hmajid2301) [reply pom.xml]: My bad missed the 0 :S

File: .gitignore [hunk 4]
```diff
@@ -28,4 +28,4 @@ target
 
 
 .history
-.vsocde
\ No newline at end of file
+.vsocde
```
Comment (pvorb) [inline .gitignore]: Please change this to `.vscode`

**Pull request merged**